### PR TITLE
Setup the code for algorithm development

### LINKED ISSUE #6: Task 1.2: Let the model predict the price after 2 weeks
Established a code example to support the prediction we want.

Please see this branch for details: https://github.com/jingedawang/StockPredictor/tree/setup/algorithm
There are still some TODOs need to be solved in the code.

### PR DESCRIPTION
**Pull request by jingedawang**

### CODE REVIEW

File: stock_predictor/data_handler.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from qlib.contrib.data.handler import Alpha158
+
+class Alpha158TwoWeeks(Alpha158):
+    
+    """
+    Override the get_label_config method in Alpha158.
+    This will build a data handler that labels the items in different way.
+    Here we labels the item with the price increase within next 10 days.
```
Comment (SwamyNallamalli) [inline stock_predictor/data_handler.py]: NIT: Will label the items with price increase in next 10 trading days.

Comment (jingedawang) [reply stock_predictor/data_handler.py]: Thanks for the advice. I have revised the sentence.
By the way, what's the meaning of NIT?

Comment (SwamyNallamalli) [reply stock_predictor/data_handler.py]: https://en.wikipedia.org/wiki/Nitpicking#:~:text=Nitpicking%20is%20a%20term%2C%20first%20used%20in%201956%2C,lice%2C%20generally%20head%20lice%29%20from%20another%20person%27s%20hair.

Nitpicking is a term that describes the action of giving too much attention to unimportant detail.

I know its not important but still commented on it :)

**Review by SwamyNallamalli — APPROVED** [review]
I am not an expert, but I felt the code is safe to go.

File: stock_predictor/data_handler.py (hunk 1 above)
Comment (jingedawang) [reply stock_predictor/data_handler.py]: Got it! I'm very happy to see any little suggestions. Each small change will make the code better. Please feel free to add comments. Thank you!

**Pull request merged**